### PR TITLE
Venues

### PR DESCRIPTION
**Pull request by zachlp2016**
Finished iteration 2 together.

### CODE REVIEW

File: lib/games.rb [hunk 1]
```diff
@@ -116,7 +116,26 @@ def average_goals_game
...
+        away_wins += 1
+      else
+        ties += 1
+      end
+    end
+    (home_wins / @stats.count.to_f * 100).round(2)
+  end
 
+  def away_wins_percent
+    100 - home_wins_percent
```
Comment (nagerz) [inline lib/games.rb]: Actually this isn't right when there are ties to consider. Can do something similar to above, maybe with some shortcuts.

File: test/games_test.rb [hunk 2]
```diff
@@ -58,6 +58,16 @@ def test_it_can_find_the_largest_difference_in_scores
     assert_equal 3, @games.blowout
   end
 
+  def test_it_can_list_away_game_scores
```
Comment (nagerz) [inline test/games_test.rb]: We missed these tests before so we added them

File: lib/games.rb [hunk 3]
```diff
@@ -101,6 +101,19 @@ def average_goals(stat)
...
+  def average_goals(stat)
+    goals_collection = {}
+    stat_collect(stat).each do |collection|
+      collection = collection.to_sym
+      goals_collection[collection] = 0
+      average = total_goals(stat)[collection] / stat_games_count(stat)[collection].to_f
+      goals_collection[collection] = average
+    end
+    goals_collection
+  end
```
Comment (nagerz) [inline lib/games.rb]: Take a look at this method. Pretty clean.

File: lib/games.rb [hunk 4]
```diff
@@ -101,6 +101,19 @@ def average_goals(stat)
     goals_collection
   end
 
+  def average_goals(stat)
+    goals_collection = {}
+    stat_collect(stat).each do |collection|
+      collection = collection.to_sym
+      goals_collection[collection] = 0
```
Comment (nagerz) [inline lib/games.rb]: Do we actually need this line?

File: test/games_test.rb [hunk 5]
```diff
@@ -79,19 +89,23 @@ def test_it_can_determine_venue_with_fewest_games
...
-    assert_equal hash, @games.average_goals_season(:season)
+    assert_equal hash, @games.average_goals_stat(:season)
   end
 
   def test_it_can_average_goals_per_game
     assert_equal 5.83, @games.average_goals_game
   end
 
+  def test_it_can_count_all_games
+    assert_equal 6, @games.stats.count
```
Comment (nagerz) [inline test/games_test.rb]: Unnecessary test really. But we found (and used in a couple methods) that just @stats.count was a real easy way to get the count of total games played.

**Comment by nagerz** [comment]
I reviewed and added a couple comments for Peregrine to check out before merging.

File: lib/games.rb (hunk 4 above)
Comment (PeregrineReed) [reply lib/games.rb]: I don't think we do, we can try it without and see if it works.

File: lib/games.rb (hunk 3 above)
Comment (PeregrineReed) [reply lib/games.rb]: It looks good! I've been thinking that using the term 'collection' for the elements is misleading though, can we revise 'collection' to 'element' instead? I'm open to other ideas for what to call it. At the point of reference in the block on line 106, we're referencing each single element in our collection.

**Pull request merged**